docker-compose-localdev.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.1'
+
+services:
+
+  wordpress:
+    volumes:
+      - ./wordpress:/var/www/html
+      - .:/var/www/html/wp-content/plugins/gutenberg
+      - ./test/e2e/test-plugins:/var/www/html/wp-content/plugins/gutenberg-test-plugins
+      - ./test/e2e/test-mu-plugins:/var/www/html/wp-content/mu-plugins
+
+  cli:
+    image: wordpress:cli
+    volumes:
+      - ./wordpress:/var/www/html
+      - .:/var/www/html/wp-content/plugins/gutenberg
+
+  wordpress_e2e_tests:
+    volumes:
+      - ./wordpress:/var/www/html
```

Comment:
For the e2e tests, I think we'd want to use the clean WordPress install, rather than something that could have any number of unknown plugins running in it?

Reply:
I thought this would be a nice way of being able to test e2e behaviour when running any number of unknown plugins, could be a way of automating plugin compatibility checks?

Reply:
I don't think it'll actually use any plugins installed. The `pretest-e2e` script runs `install-wordpress.sh --e2e_tests`, which resets the DB.

Reply:
Ahhh, well in that case, that configuration's gone!

Still, would be nice to have that option in the future, a plugin compatibility checker would be _sweeeeet_